impala/assets/dashboards/impala_catalog_overview.json

```diff
@@ -0,0 +1,1033 @@
+          {
+            "id": 4446320522723288,
+            "definition": {
+              "type": "note",
+              "content": "This is a high-level overview of your Impala daemons, so you can track health status, search and indexing performance, and resource utilization metrics from all your services and be better prepared to address potential issues.\n\n##### [**Datadog Impala integration docs&nbsp;↗**](https://docs.datadoghq.com/integrations/impala/)",
+              "background_color": "transparent",
+              "font_size": "18",
+              "text_align": "left",
+              "show_tick": false,
+              "tick_pos": "50%",
+              "tick_edge": "left",
+              "has_padding": true
+            },
+            "layout": {
+              "x": 0,
+              "y": 0,
+              "width": 6,
+              "height": 2
+            }
+          }
+        ]
+      },
+      "layout": {
+        "x": 0,
+        "y": 0,
+        "width": 6,
+        "height": 5
+      }
+    },
+    {
+      "id": 7311366610321510,
+      "definition": {
+        "title": "Overview",
+        "title_align": "left",
+        "type": "group",
+        "background_color": "blue",
+        "show_title": true,
+        "layout_type": "ordered",
+        "widgets": [
+          {
+            "id": 4051455317781594,
+            "definition": {
+              "title": "Services up",
+              "title_size": "16",
+              "title_align": "left",
+              "type": "check_status",
+              "check": "impala.catalog.openmetrics.health",
+              "grouping": "cluster",
+              "group_by": [],
+              "tags": []
+            },
+            "layout": {
+              "x": 0,
+              "y": 0,
+              "width": 6,
+              "height": 2
+            }
+          },
+          {
+            "id": 148563693988418,
+            "definition": {
+              "type": "note",
+              "content": "The catalog is not critical to the normal operation of an Impala cluster. \n\nProblems with those daemons do not result in data loss.",
+              "background_color": "blue",
+              "font_size": "14",
+              "text_align": "left",
+              "vertical_align": "center",
+              "show_tick": true,
+              "tick_pos": "50%",
+              "tick_edge": "top",
+              "has_padding": true
+            },
+            "layout": {
+              "x": 0,
+              "y": 2,
+              "width": 6,
+              "height": 2
+            }
+          }
+        ]
+      },
+      "layout": {
+        "x": 6,
+        "y": 0,
+        "width": 6,
+        "height": 5
+      }
+    },
+    {
+      "id": 357184664471320,
+      "definition": {
+        "title": "System",
+        "title_align": "left",
+        "type": "group",
+        "background_color": "pink",
+        "show_title": true,
+        "layout_type": "ordered",
+        "widgets": [
```

Comment:
Some of the memory charts include a nice percentage widget on the right which looks pretty nice, others don't. Something similar happens with threads. Would it make sense to use it consistently everywhere?

Reply:
Yes I added some value widgets from time to time. My idea was to show off what's possible to be done with the dashboards, that's why it's not consistent. I can try to add more if you think they look cool!